src/ssl.c

```diff
@@ -5910,6 +6040,11 @@ int wolfSSL_CertManagerVerifyBuffer(WOLFSSL_CERT_MANAGER* cm, const byte* buff,
     if (ret == 0)
         ret = ParseCertRelative(cert, CERT_TYPE, 1, cm);
 
+    /* if verify callback has been set */
```

Comment:
The calls to `DoVerifyCallback` perform the callback after the OCSP/CRL checking. Perhaps this callback should be below those?

Reply:
Agreed.